frontend/src/Maker.tsx

```diff
@@ -31,7 +31,14 @@ interface CfdSellOrderPayload {
 }
 
 async function postCfdSellOrderRequest(payload: CfdSellOrderPayload) {
-    let res = await fetch(`/api/order/sell`, { method: "POST", body: JSON.stringify(payload), credentials: "include" });
+    let res = await fetch(`/api/order/sell`, {
+        method: "POST",
+        body: JSON.stringify(payload),
+        headers: {
+            "Content-Type": "application/json",
+        },
```

Comment:
out of curiosity: would this header be needed if we stayed with `axiom` after adding credentials? 
(I'm trying to figure out the pros and cons of each)